hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/lifecycle/OMLifecycleConfigurationCreateRequest.java

```diff
@@ -0,0 +1,169 @@
+import org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos.OMResponse;
+import org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos.UserInfo;
+import org.apache.hadoop.ozone.security.acl.IAccessAuthorizer;
+import org.apache.hadoop.ozone.security.acl.OzoneObj;
+import org.apache.hadoop.util.Time;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handles CreateLifecycleConfiguration Request.
+ */
+public class OMLifecycleConfigurationCreateRequest extends OMClientRequest {
+  private static final Logger LOG =
+      LoggerFactory.getLogger(OMLifecycleConfigurationCreateRequest.class);
+
+  public OMLifecycleConfigurationCreateRequest(OMRequest omRequest) {
+    super(omRequest);
+  }
+
+  @Override
+  public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
+    CreateLifecycleConfigurationRequest request =
+        getOmRequest().getCreateLifecycleConfigurationRequest();
+    LifecycleConfiguration lifecycleConfiguration =
+        request.getLifecycleConfiguration();
+
+    OmUtils.validateVolumeName(lifecycleConfiguration.getVolume(), ozoneManager.isStrictS3());
+    OmUtils.validateBucketName(lifecycleConfiguration.getBucket(), ozoneManager.isStrictS3());
+
+    CreateLifecycleConfigurationRequest.Builder newCreateRequest =
+        request.toBuilder();
+
+    LifecycleConfiguration.Builder newLifecycleConfiguration =
+        lifecycleConfiguration.toBuilder();
+
+    newLifecycleConfiguration.setCreationTime(Time.now());
+    newCreateRequest.setLifecycleConfiguration(newLifecycleConfiguration);
+
+    return getOmRequest().toBuilder().setUserInfo(getUserInfo())
+        .setCreateLifecycleConfigurationRequest(newCreateRequest.build())
+        .build();
+  }
+
+  @Override
+  public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, ExecutionContext context) {
+    final long transactionLogIndex = context.getIndex();
+    OMMetadataManager metadataManager = ozoneManager.getMetadataManager();
+
+    CreateLifecycleConfigurationRequest createLifecycleConfigurationRequest =
+        getOmRequest().getCreateLifecycleConfigurationRequest();
+
+    LifecycleConfiguration lifecycleConfiguration =
+        createLifecycleConfigurationRequest.getLifecycleConfiguration();
+
+    String volumeName = lifecycleConfiguration.getVolume();
+    String bucketName = lifecycleConfiguration.getBucket();
+
+    OMResponse.Builder omResponse = OmResponseUtil.getOMResponseBuilder(
+        getOmRequest());
+
+    AuditLogger auditLogger = ozoneManager.getAuditLogger();
+    UserInfo userInfo = getOmRequest().getUserInfo();
+
+    String bucketKey = metadataManager.getBucketKey(volumeName, bucketName);
+    IOException exception = null;
+    boolean acquiredBucketLock = false;
+    OMClientResponse omClientResponse = null;
+    Map<String, String> auditMap = new HashMap<>();
+
+    try {
+      if (ozoneManager.getAclsEnabled()) {
```

Comment:
After  HDDS-9388, ACL check is moved to preExecute(). You can refer to OMSnapshotCreateRequest.